semperfiwebdesign/all-in-one-seo-pack
# Fix homepage showing latest posts doesn't show up in sitemap

Issue #2757

## Proposed changes

Fixes a bug where the homepage doesn't show up in the sitemap when it's showing the latest posts.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist




## Testing instructions

1. Make sure the homepage is set to show the latest posts under Reading.
2. Make sure Sitemap Indexes are enabled.
3. Make sure that the homepage shows up in the addl sitemap file.
